R/plot_repos.R

```diff
@@ -11,15 +11,11 @@
 plot_repos <- function(gitstats_obj,
                        repos_n = 10) {
   fullname <- organization <- name <- last_activity_at <- NULL
-  if (is.null(gitstats_obj$repos_dt)) {
-    stop("You have to first construct your repos data.frame with 'get_repos' function.",
-      call. = FALSE
-    )
-  }
 
-  repos_dt <- gitstats_obj$repos_dt
+  repos_dt <- gitstats_obj$show_repos()
 
-  repos_to_plot <- head(repos_dt, repos_n)
+  repos_to_plot <- repos_dt[order(last_activity_at)]
```

Comment:
While going through example_workflow.R file the code crashes in this line and I'm not able to plot repos. It looks like it has some issues with the way you declare sorting of the table.

![image](https://user-images.githubusercontent.com/32685910/235129024-cc8d0e4e-d22e-4766-85d1-59d73538a7b9.png)

Reply:
I get this error when the input to plot function is empty (so when I try to plot repos before pulling them to `GitStats`) - can you check you've pulled first repos? 
I pushed NULL handler to plots right now - so when the repos/commits are empty, the error pops out with the appropriate message.

Reply:
After some changes I was able to print plot. Looks good but if I could suggest something: can we remove the name of the organization on the Y axis? It's doubled information with the color: 
![image](https://user-images.githubusercontent.com/32685910/235138934-811ecc97-6686-49ed-9f96-96a2b3125a99.png)

It's not critical though.